.github/workflows/release-image-ecr.yaml

```diff
@@ -4,11 +4,18 @@ on:
   push:
     branches:
```

Comment:
Are the conditions must be satisfied together?

Reply:
no,It's OR logical